widgetssdk/src/main/java/com/glia/widgets/view/textview/ChatInputHintFix.kt

```diff
@@ -0,0 +1,87 @@
+package com.glia.widgets.view.textview
+
+import android.content.Context
+import android.text.Editable
+import android.text.TextUtils
+import android.text.TextWatcher
+import android.util.AttributeSet
+
+/**
+ * This class is here to solve next bug:
+ * See video in this comment to understand initial bug: https://glia.atlassian.net/browse/MOB-3860?focusedCommentId=459636
+ *
+ * The issue was caused by default EditText behavior. The hint height is effecting the EditText height even when the hint is not displayed.
+ * This means that of for some reason your EditText view has property maxLines = 4 and your hint is so long that it takes 4 lines and
+ * then when user will wright just 1 letter into your EditText it would still as much space as if it had 4 lines of code.
```

Comment:
Let me suggest several improvements to the comment:
of for some reason => if for some reason
will wright => 'will write' or just 'writes'
it would still as much space => it would still occupy as much height

Reply:
Fixed in ef93d61